int32 path_len
inof_robot/Pose[] points
